src/utils/neurons.utils.spec.ts

```diff
@@ -68,22 +69,22 @@ describe("neurons-utils", () => {
     expect(ineligible.length).toEqual(0);
   });
 
-  it("should not have not voted neurons because ineligible", () => {
-    let notVoted = notVotedNeurons({
+  it("should not have votable neurons because ineligible", () => {
+    let notVoted = votableNeurons({
```

Comment:
should we rename the variable `notVoted` too?

Reply:
Done 👍